### PR TITLE
Workaround ykjit/yk#537.

### PR DESCRIPTION
**Pull request by vext01**
Needed to fix breakage introduced by: https://github.com/ykjit/ykllvm/pull/33

Merge the other PR first.

### CODE REVIEW

File: tests/tests/blockmap.c [hunk 1]
```diff
@@ -18,12 +18,16 @@ int main(int argc, char **argv) {
 
 // This isn't used as part of the test, but is required for this file to
 // compile with ykllvm.
-void unused() {
+//
+// FIXME: This only returns an integer due to an shortcoming of the stopgap interpreter:
```
Comment (ltratt) [inline tests/tests/blockmap.c:22]: s/an shortcoming/a shortcoming/

Comment (vext01) [reply tests/tests/blockmap.c]: Oops. Fixed in 71df7ea

**Comment by ptersilie** [comment]
Looks good.

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat.

**Pull request merged**